batch.go

```diff
@@ -1101,6 +1101,8 @@ func (b *Batch) upsertOffsets() error {
 		debitED.TransactionCode = CheckingDebit
 	case OffsetSavings:
 		debitED.TransactionCode = SavingsDebit
+	default:
+		return fmt.Errorf("unknown offset account type: %s", b.offset.AccountType)
```

Comment:
Should these be `var` and declared somewhere else?

Reply:
A `var` so we can compare against it a bit better? We can do that.